js/ui/shared/grouped_data_converter_mixin.js

```diff
@@ -28,6 +35,7 @@ export default {
                     type: "array",
                     data: dataSource
                 },
+                searchExpr: "text",
```

Comment:
use searchExpr: "text", only for non-objects

Reply:
Sure